### ISSUE TITLE
[Items] Trigger socketed spell is not parsed

### ISSUE THREAD

**Issue by leMicin** [domialex/Sidekick#430]
```
Rarity: Rare
Vortex Weaver
Vaal Sceptre
--------
Sceptre
Physical Damage: 37-70
Elemental Damage: 2-43 (augmented)
Critical Strike Chance: 6.00%
Attacks per Second: 1.40
Weapon Range: 11
--------
Requirements:
Level: 64
Str: 113
Int: 113
--------
Sockets: R-B 
--------
Item Level: 80
--------
32% increased Elemental Damage (implicit)
--------
Adds 2 to 43 Lightning Damage
Adds 14 to 27 Fire Damage to Spells
5% increased Cast Speed
+5 Mana gained on Kill
Trigger a Socketed Spell when you Use a Skill (crafted)
```

**Comment by leMicin** [comment]
The text from the API is Chance to trigger a socketed spell when you use a skill. This is similar to #422 where the text are very different. Do we start adding a dictionary between what the API says and what the game says? I would hope to not have to do that.

**Comment by JadedCricket** [comment]
Seems this problem with certain stats whose wording change under specific circumstances.

Instances that I've confirmed are:
- '_#% chance to Trigger a Socketed Spell when you Use a Skill_'. At 100% it is '_Trigger a Socketed Spell when you Use a Skill_'
- '_#% chance to Avoid being Frozen_'. At 100% it is '_Cannot be Frozen_'
- '_#% reduced Mana Cost of Skills_'. Greater than 0% it is '_#% increased Mana Cost of Skills_'

There are probably several instances of that last one. I'm not sure how we would solve this problem without some kind of mapping unless the POE API provides that for us, which I doubt.

**Comment by domialex** [comment]
Same with `Projectiles Pierce # additional Targets` and `Projectiles Pierce an additional Target`

**Comment by leMicin** [comment]
Fixed in 2.0
